charts/kafka-operator/values.yaml

```diff
@@ -34,6 +34,9 @@ operator:
 
 webhook:
   enabled: true
+  serverPort:
```

Comment:
if we don't provide a default value here I'd just leave this field commented out like # serverPort